### ISSUE TITLE
during training, why so many "-1" in the training log?

### ISSUE THREAD

**Issue by xiaoxiongli** [zhaoweicai/cascade-rcnn#58]
I0920 19:51:39.189030 29984 sgd_solver.cpp:105] Iteration 5200, lr = 0.001
I0920 19:53:27.133998 29984 solver.cpp:219] Iteration 5300 (0.926367 iter/s, 107.949s/100 iters), loss = 0.495927
I0920 19:53:27.134055 29984 solver.cpp:238]     Train net output #0: bbox_iou = 0.654935
I0920 19:53:27.134068 29984 solver.cpp:238]     Train net output #1: bbox_iou_2nd = 0.703076
I0920 19:53:27.134075 29984 solver.cpp:238]     Train net output #2: bbox_iou_3rd = 0.776575
I0920 19:53:27.134081 29984 solver.cpp:238]     Train net output #3: bbox_iou_pre = 0.56266
I0920 19:53:27.134088 29984 solver.cpp:238]     Train net output #4: bbox_iou_pre_2nd = 0.720895
I0920 19:53:27.134091 29984 solver.cpp:238]     Train net output #5: bbox_iou_pre_3rd = 0.769399
I0920 19:53:27.134097 29984 solver.cpp:238]     Train net output #6: cls_accuracy = 0.974609
I0920 19:53:27.134101 29984 solver.cpp:238]     Train net output #7: cls_accuracy_2nd = 0.958984
I0920 19:53:27.134107 29984 solver.cpp:238]     Train net output #8: cls_accuracy_3rd = 0.947266
I0920 19:53:27.134117 29984 solver.cpp:238]     Train net output #9: loss_bbox = 0.0370498 (* 1 = 0.0370498 loss)
I0920 19:53:27.134125 29984 solver.cpp:238]     Train net output #10: loss_bbox_2nd = 0.059268 (* 0.5 = 0.029634 loss)
I0920 19:53:27.134136 29984 solver.cpp:238]     Train net output #11: loss_bbox_3rd = 0.0315293 (* 0.25 = 0.00788233 loss)
I0920 19:53:27.134143 29984 solver.cpp:238]     Train net output #12: loss_cls = 0.060164 (* 1 = 0.060164 loss)
I0920 19:53:27.134153 29984 solver.cpp:238]     Train net output #13: loss_cls_2nd = 0.126871 (* 0.5 = 0.0634354 loss)
I0920 19:53:27.134160 29984 solver.cpp:238]     Train net output #14: loss_cls_3rd = 0.0889007 (* 0.25 = 0.0222252 loss)
I0920 19:53:27.134169 29984 solver.cpp:238]     Train net output #15: rpn_fpn2_accuracy = 0.999974
I0920 19:53:27.134176 29984 solver.cpp:238]     **Train net output #16: rpn_fpn2_accuracy = -1**
I0920 19:53:27.134181 29984 solver.cpp:238]     **Train net output #17: rpn_fpn2_bboxiou = -1**
I0920 19:53:27.134188 29984 solver.cpp:238]     Train net output #18: rpn_fpn2_loss = 0.00119714 (* 0.25 = 0.000299286 loss)
I0920 19:53:27.134198 29984 solver.cpp:238]     Train net output #19: rpn_fpn2_loss = 0 (* 0.25 = 0 loss)
I0920 19:53:27.134204 29984 solver.cpp:238]     Train net output #20: rpn_fpn3_accuracy = 0.999059
I0920 19:53:27.134209 29984 solver.cpp:238]     **Train net output #21: rpn_fpn3_accuracy = -1**
I0920 19:53:27.134214 29984 solver.cpp:238]     **Train net output #22: rpn_fpn3_bboxiou = -1**
I0920 19:53:27.134222 29984 solver.cpp:238]     Train net output #23: rpn_fpn3_loss = 0.00280852 (* 0.25 = 0.000702129 loss)
I0920 19:53:27.134230 29984 solver.cpp:238]     Train net output #24: rpn_fpn3_loss = 0 (* 0.25 = 0 loss)
I0920 19:53:27.134236 29984 solver.cpp:238]     Train net output #25: rpn_fpn4_accuracy = 0.999842
I0920 19:53:27.134241 29984 solver.cpp:238]     **Train net output #26: rpn_fpn4_accuracy = -1**
I0920 19:53:27.134246 29984 solver.cpp:238]     **Train net output #27: rpn_fpn4_bboxiou = -1**
I0920 19:53:27.134253 29984 solver.cpp:238]     Train net output #28: rpn_fpn4_loss = 0.00663789 (* 0.25 = 0.00165947 loss)
I0920 19:53:27.134263 29984 solver.cpp:238]     Train net output #29: rpn_fpn4_loss = 0 (* 0.25 = 0 loss)
I0920 19:53:27.134269 29984 solver.cpp:238]     Train net output #30: rpn_fpn5_accuracy = 0.970713
I0920 19:53:27.134274 29984 solver.cpp:238]     Train net output #31: rpn_fpn5_accuracy = 1
I0920 19:53:27.134279 29984 solver.cpp:238]     Train net output #32: rpn_fpn5_bboxiou = 0.504063
I0920 19:53:27.134286 29984 solver.cpp:238]     Train net output #33: rpn_fpn5_loss = 0.0666939 (* 0.25 = 0.0166735 loss)
I0920 19:53:27.134296 29984 solver.cpp:238]     Train net output #34: rpn_fpn5_loss = 0.0310986 (* 0.25 = 0.00777464 loss)
I0920 19:53:27.134310 29984 sgd_solver.cpp:105] Iteration 5300, lr = 0.001

during training, why so many "-1" in the training log? is there something wrong?  How can i fix this? Please Help...
